implementations/micrometer-registry-appoptics/src/main/java/io/micrometer/appoptics/AppOpticsConfig.java

```diff
@@ -65,6 +65,13 @@ default String uri() {
         return v == null ? "https://api.appoptics.com/v1/measurements" : v;
     }
 
+    /**
+     * @return whether or not to ship a floored time - synchronizes reporting across nodes
+     */
+    default boolean floorTimes() {
+        return get(prefix() + ".floorTimes") == "true";
```

Comment:
For more lenient and conventional parsing, I would suggest the following.
```suggestion
        return Boolean.parseBoolean(get(prefix() + ".floorTimes"));
```